app/static/js/volunteerProfile.js

```diff
@@ -114,6 +114,24 @@ $(document).ready(function(){
       }
     })
   });
+  $("#updatePhone").click(function(){
+    userName=$(this).data("username") 
+    phoneNumber=$("#newinput").val()
+    let isvalid =phoneNumber.replace(/\D/g,"").length
+    if (isvalid ===10){
+      $.ajax({
+        method:"POST",
+        url:"/updatePhone",
+        data:{"username":userName,
+              "phoneNumber":phoneNumber}
+      })
+      msgFlash("Phone Number is updated", "success")  
```

Comment:
Let's use ajax success and error functions to make sure we are only flashing a success message if we were successful